maxim/tests/test_test_runs.py

```
@@ -241,5 +261,597 @@ def error(self, message: str, e: Optional[Exception] = None) -> None:
+        }
+
+        result = (
+            self.maxim.create_test_run(
+                name="test run with image variables and prompt version",
+                in_workspace_id=workspaceId,
+            )
+            .with_data_structure(data_structure)
+            .with_data(data)
+            .with_concurrency(1)  # Lower concurrency for image processing
+            .with_evaluators("Bias", "Clarity")
+            .with_prompt_version_id(promptVersionId)  # Test with prompt version
```

Ruff (0.11.9)

Undefined name `promptVersionId`

(F821)